### ISSUE TITLE
Discord update broke toolbar icon (and partial fix)

### ISSUE THREAD

**Issue by re11ding** [iyu46/RemovedConnectionAlerts#23]
Discord went and changed all their class names again! Woo! 

For anyone wanting to be able to click the icon again, look for `const CssClasses` in the code. Replacing its section with the one below should make it work again. 

```
    const CssClasses = {
        container: 'rcaHistoryContainer',
        item: 'rcaHistoryItem',
        avatar: 'rcaHistoryAvatar',
        info: 'rcaHistoryInfo',
        header: 'rcaHistoryHeader',
        logDeleteBtn: 'rcaHistoryDeleteBtn',
        logDeleteBtnIconLabel: 'rcaHistoryDeleteBtnIcon',
        logDeleteBtnIconClass: 'winButtonClose__73489 winButton__88672 rcaHistoryDeleteBtnIcon',
        settingsIconClass: 'winButtonMinMax__72f36 winButton__88672 rcaHistoryDeleteBtnIcon',
        emptyMessage: 'rcaModalEmptyMessage',
        emptyMessageText: 'rcaModalEmptyMessageText',
        recentsIcon: 'recentsIcon_a585d7',
        toolbarIcon: 'iconWrapper_de6cd1 clickable_ce0925',
        voiceButton: 'button__6d958',
    };
```

Use this for the time being until a proper update is created!

**Comment by iyu46** [comment]
Thanks for letting me know! I'm on it

**Comment by iyu46** [comment]
Fixed in the newest update, 0.8.0! Also in this commit where I messed up referring to this ticket in the comment https://github.com/iyu46/RemovedConnectionAlerts/commit/d35ef3a68646b1f1e6294a691a89e62bfa3eb2f0
